client/src/app/core/ui-services/base-filter-list.service.ts

```diff
@@ -104,7 +104,10 @@ export abstract class BaseFilterListService<M extends BaseModel, V extends BaseV
     /**
      * Constructor.
      */
-    public constructor(private store: StorageService, private repo: BaseRepository<V, M>) {}
+
+    public constructor(private store: StorageService, private repo: BaseRepository<V, BaseModel>) {
+        // TODO
```

Comment:
What is to do here?

Reply:
BaseModel should be of the same kind as the viewModel (V)

Reply:
This cannot be done currently through generics. I'll create an issue for this, so please remove the todo there. (#4596)